compaction/compaction_manager.cc

```diff
@@ -264,10 +264,10 @@ compaction_manager::compaction_state& compaction_manager::get_compaction_state(c
     }
 }
 
-compaction_manager::task::task(compaction_manager& mgr, replica::table* t, sstables::compaction_type type, sstring desc)
+compaction_manager::task::task(compaction_manager& mgr, compaction::table_state* t, sstables::compaction_type type, sstring desc)
```

Comment:
Again, we could switch to a reference here, if we are touching this code anyway.

Reply:
(also, the compaction:: qualifier seems extraneous but in no way harmful.

Reply:
Let's defer this kind of changes. The series is long enough already.